### ISSUE TITLE
Proper name for kernel options/parameters

### ISSUE THREAD

**Issue by sknorr** [SUSE/doc-styleguide#29]
So, for one, we need a general preference for either "option" or "parameter" when it comes to the command line.

Second, we need to find the right combination of "kernel + [boot]? + option/parameter".

**Comment by sknorr** [comment]
`docfight -i` in doc-sle gives me this:

* 46 x kernel parameter
* 8 x kernel option
* 6 x kernel command line parameter
* 2 x kernel boot parameter
* 0 x kernel boot option

There is a clear favorite here.

**Comment by cwickert** [comment]
I agree we should standardize here, however I think there is still a semantic difference between option and parameter in this context. For example, a networked boot is an _option_ (one way of doing it) but requires one or more _parameters_.

**Comment by dariavladykina** [comment]
There are 2 things here:
1. Defining `option` and `parameter` themselves (great explanation [here](https://stackoverflow.com/questions/36495669/difference-between-terms-option-argument-and-parameter))
2. Standardizing the combo `boot option` and `kernel parameter` (what else) because they seem to be in question in this issue.

**Comment by jfaltenbacher** [comment]
also the docbook tag allowlist gives some guidance on `option` and `parameter`:
`option`: An option for a software command
`parameter`: A value or a symbolic reference to a value

**Comment by dariavladykina** [comment]
Committee has decided on selecting 2 terms e.g. boot option and kernel parameter, defining them and gradually cleaning up/updating the documentation.

**Comment by dariavladykina** [comment]
For "boot option", found this definition in [Microsoft Learn](https://learn.microsoft.com/en-us/windows-hardware/drivers/devtest/boot-options-for-driver-testing-and-debugging):
_Boot options are values that determine how the boot loader loads and configures the operating system and other bootable programs and devices_.

For "kernel parameter", these are from RH and Ubuntu respectively: 

_- Kernel parameters are tunable values which you can adjust while the system is running. 
- Kernel boot parameters are text strings which are interpreted by the system to change specific behaviors and enable or disable certain features._

**Comment by dariavladykina** [comment]
Added to TermWeb.
